hail/src/main/scala/is/hail/HailContext.scala

```diff
@@ -385,14 +382,14 @@ object HailContext {
 }
 
 class HailContext private(val sc: SparkContext,
-  val sqlContext: SQLContext,
   val logFile: String,
   val tmpDir: String,
   val branchingFactor: Int,
   val optimizerIterations: Int) {
   val hadoopConf: hadoop.conf.Configuration = sc.hadoopConfiguration
   val sHadoopConf: SerializableHadoopConfiguration = new SerializableHadoopConfiguration(hadoopConf)
   val hadoopConfBc: Broadcast[SerializableHadoopConfiguration] = sc.broadcast(sHadoopConf)
+  val sparkSession = SparkSession.getActiveSession.get
```

Comment:
looks like this doesn't work

Reply:
software is terrible
> Apparently there is no way how to initialize SparkSession from existing SparkContext.

https://stackoverflow.com/a/43372817/6823256

Reply:
Judging by the Spark library code, we can just set the config from the spark context and say getOrCreate and it will find the existing spark context.